src/vic.c

```diff
@@ -1,18 +1,22 @@
 #include "vic.h"
 
-char *vic_err_msg[] = {
-    /* VIC_ERR_NO */
-    "",
+/* VIC_ERR_NO */
+const char _err_msg_0[] PROGMEM = "";
 
-    /* VIC_ERR_INSUFFICIENT_SPACE */
-    "ERROR: No memory for new variable or function\n",
+/* VIC_ERR_INSUFFICIENT_SPACE */
+const char _err_msg_1[] PROGMEM = \
+    "ERROR: No memory for new variable or function\n";
 
-    /* VIC_ERR_INVALID_NAME */
-    "ERROR: Invalid name\n",
+/* VIC_ERR_INVALID_NAME */
+const char _err_msg_2[] PROGMEM = \
+    "ERROR: Invalid name\n";
 
-    /* VIC_ERR_INVALID_ARGS */
-    "ERROR: Arguments are of invalid type or there is wrong count of them\n"
-};
+/* VIC_ERR_INVALID_ARGS */
+const char _err_msg_3[] PROGMEM = \
+    "ERROR: Arguments are of invalid type or there is wrong count of them\n";
+
+const char * const vic_err_msg[] PROGMEM = {_err_msg_0, _err_msg_1,
```

Comment:
@vargac This is just a nitpick but could we put these error messages on separate lines? Like

```c
const char * const vic_err_msg[] PROGMEM = {
  _err_msg_0,
  _err_msg_1,
  _err_msg_2,
  _err_msg_3
}
```